.gitignore

```diff
@@ -1,3 +1,24 @@
 bin/
 dist/
 /.vscode/
+
+## VIM
```

Comment:
Shouldn’t all that be in your global gitignore on your machine?

Reply:
Sure, that will work as well. Putting those in project-specific configs is force of habit for me.